src/ape/contracts/base.py

```diff
@@ -185,6 +185,19 @@ class ContractEvent:
 
 
 class ContractInstance(AddressAPI):
+    """
+    An instance of a smart contract that you can interact with.
+    After you deploy a contract using the :class:`~ape.api.accounts.AccountAPI.deploy` method,
+    you get back a contract instance.
+
+    Usage example::
```

Comment:
why do we :: ?

Reply:
That is required to make it show up as the code-block style.